www/js/app.js

```diff
@@ -631,31 +631,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         apiStatusPanel.classList.remove('card-success', 'card-warning', 'card-danger');
         var apiPanelClass = 'card-success';
         if (isMessageChannelAvailable()) {
-            $('#messageChannelStatus').html("MessageChannel API available");
-            $('#messageChannelStatus').removeClass("apiAvailable apiUnavailable")
-                    .addClass("apiAvailable");
+            let messageChannelStatus = document.getElementById('messageChannelStatus');
+            messageChannelStatus.innerHTML = 'MessageChannel API available';
+            messageChannelStatus.classList.remove('apiAvailable', 'apiUnavailable');
+            messageChannelStatus.classList.add('apiAvailable');
         } else {
             apiPanelClass = 'card-warning';
-            $('#messageChannelStatus').html("MessageChannel API unavailable");
-            $('#messageChannelStatus').removeClass("apiAvailable apiUnavailable")
-                    .addClass("apiUnavailable");
+            let messageChannelStatus = document.getElementById('messageChannelStatus');
```

Comment:
If you do as mentioned in previous comment, you wouldn't need this.

Reply:
Correct.